test/base/browser_tests_main.cc

```diff
@@ -31,6 +37,12 @@ int main(int argc, char** argv) {
   }
 #endif  // BUILDFLAG(IS_WIN)
 
+#if BUILDFLAG(ENABLE_PLAYLIST)
+  base::ScopedAddFeatureFlags scoped_feature_flags(
+      base::CommandLine::ForCurrentProcess());
+  scoped_feature_flags.DisableIfNotSet(playlist::features::kPlaylist);
```

Comment:
this looks weird. what are we trying to do here? feature should be disabled anyway, because it's disabled by default.